_posts/2022-12-12-trino-summit-2022-starburst-recap.md

```diff
@@ -0,0 +1,87 @@
+[Trino Summit 2022 recap post series]({% post_url
+2022-11-21-trino-summit-2022-recap %}). Because I have a perpetual fear of
+missing out, I convinced the two of them that I should get in on the fun.
+For this latest installment in the series, I will be recapping my very own Trino
+Summit talk. Basically, I'm ripping off Bo Burnham's comedy bit
+where he [reacts to his own reaction video](https://youtu.be/FZVMB8mrNO0?t=35),
+blog style.
+
+In this session, I demonstrate building a data lakehouse architecture with
+Starburst Galaxy, the fastest and easiest way to get up running with Trino.
+Before I dive into the recap, I want to thank the Trino community for showing
+up. I am grateful that I was able to meet and learn from so many members of the
+community in person.
+
+<!--more-->
+
+{% youtube Zfmxwu0m98k %}
+
+
+## Recap
+
+The premise of this example is that we have Pokemon Go data being ingested into
+S3, which contains encounter information of each Pokemon. This includes the
+geo-location data, aka the latitude and longitude, of where each Pokemon spawned
+and how long the Pokemon was at that location. What we don’t have is any
+information on that Pokemon’s abilities, that’s all contained in the Pokédex
+stored in MongoDB which I’ve cleverly nicknamed **PokeMongoDB**. This has all
+the statistics on your desired Pokemon including type, legendary status,
+catch rate, and more. To make meaningful information out of our data, we need to
+combine the incoming geo-location data with the static dimension CSV table
+located in MongoDB.
+
+To do this, I build out a reporting structure in the data lake using
+Starburst Galaxy. The first step is to read the data in the land layer, then
+clean and optimize that data into more performant orc files in the structure
+layer. Finally, I join the spawn data & Pokédex data together into a single
+table that is cleaned and ready to be utilized by a data consumer. Since I only
+want account administrators to have access to update the consume layer, I
+demonstrate applying role-based access control capabilities within Starburst
+Galaxy, which provides the proper data governance to satisfy this request. I
+then create some visualizations to analyze which Pokemon are common to spawn in
+the San Francisco Bay Area.
+
+I walk through all the setup required to put this data lakehouse architecture
+into action including creating my catalogs, clusters, schemas, and tables. 
+After incorporating open-table formats, applying native security, and building
+out a reporting structure, I have confidence that my data lakehouse is built
+to last (and end up with some really cool final Pokemon graphs). I'm working on
+turning this demo into a tutorial, and will update the post when this happens.
+In the mean time, if you are curious about trying
+Starburst Galaxy, I invite you to look at one of the [already created tutorials](https://docs.starburst.io/starburst-galaxy/tutorials/index.html).
+
+
+## Datanova 2023
```

Comment:
I think this section is not appropriate for the individual recap post. We could add something like this in the final post with the teaser/fomo video as an outlook on whats next when we talk about next Trino summit. But here.. I would remove. Ultimately however I think @martint should make the call in this instance.

Reply:
Agreed, let's defer to Martin.